src/explore-education-statistics-common/src/modules/charts/components/CustomTooltip.tsx

```diff
@@ -60,8 +64,8 @@ const CustomTooltip = ({
 
                     <strong>
                       {formatPretty(
-                        item.value.toString(),
-                        item.unit,
+                        (item.value as number | string).toString(),
+                        item.unit as string,
```

Comment:
The unit is typed as a `ReactNode`, meaning it could be practically anything. Again, we should try to handle this to avoid future bugs. Something like this should probably be sufficient:

```ts
typeof item.unit === 'string' ? item.unit : ''
```